src/conq/clients.py

```diff
@@ -21,3 +22,4 @@ class Clients:
     raycast: Optional[RayCastClient]=None
     command: Optional[RobotCommandClient]=None
     robot: Optional[Robot]=None
+    world_object: Optional[WorldObjectClient]=None
```

Comment:
You had mentioned you might remove this. Just wanted to remind you of that if you don't need it anymore.

Reply:
I went through my code in `tool_detector_interface.py` and realized I do have some examples of adding world objects to the map. What do you think about keeping these as a potential resource since tool detector will mostly be an example/WIP?

Reply:
I think that's a good call. Leaving this stuff in isn't going to hurt anything so I say keep it